src/handlers/createFile.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from 'fs';
+import * as mkdirp from 'mkdirp';
+import * as path from 'path';
+import * as ejs from 'ejs';
+import {isUndefined} from 'lodash';
+import CreateFileInfo from '../instructions/CreateFileInfo';
+
+export default function createFile({
+  content,
```

Comment:
please name this parameter params and add render and reporter like how `appendFile` does.